docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+---
+aliases:
+  - ../basics/timeseries/
+description: Introduction to Prometheus
+keywords:
+  - grafana
+  - intro
+  - Prometheus
+  - metrics
+  - time series
+title: What is Prometheus?
+weight: 300
+---
+
+# What is Prometheus?
+
+Observability focuses on understanding the internal state of systems based on the data they produce, which helps determine if the infrastructure is operating as expected. Prometheus is a core technology for system observability. However, Prometheus can be a confusing term because it is used in different contexts. Understanding Prometheus basics, why it’s valuable, and how people use it in practice will help you use Grafana.
+
+Prometheus was started in 2012 at SoundCloud, who found existing technologies such as Graphite insufficient for their observability needs. Prometheus has a robust data model and query language, both of which are addressed in the following sections, and is simple and scalable. In 2018, Prometheus graduated from Cloud Native Computing Foundation (CNCF) incubation, and today has a thriving community.
+
+## Prometheus as data
+
+The following panel in a Grafana dashboard shows how much disk bandwidth on a Mac laptop is being used. The green line represents disk `reads`, and the yellow line represents `writes`.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/disk-io.png" max-width="750px" caption="Disk I/O dashboard" >}}
+
+Data like these form _time series_. The X-axis is a moment in time, and the Y-axis is a number or measurement, for example, 5 megabytes per second. This kind of time series data appears everywhere in systems monitoring, but also in places like seasonal temperature charts and stock prices. This data is simply some measurement (such as the Tesla stock price, or Disk I/O) through a series of time instants.
+
+Prometheus is a technology used to collect and store time series data. Time series are fundamental to Prometheus; its [data model](https://prometheus.io/docs/concepts/data_model/) is arranged into:
+
+- _metrics_ that consist of a _timestamp_ and a _sample_, which is the numeric value, such as how many disk bytes have been read or a stock price
+- a set of labels called _dimensions_
+
+You can store time series data in any relational database, but these kinds of systems are not developed to store and query large volumes of time series data. Prometheus and similar software provide tools to compact and optimize time series data.
+
+### Example
+
+The following Grafana dashboard image shows a disk IO graph of raw data from Prometheus that is derived from a laptop.
+
+The **Metrics browser** field contains the following query:
+
+`node_disk_written_bytes_total{job=”integrations/macos-node”, device!=””}`
```

Comment:
minor - note use of smart quotes ” here rather than regular " ascii.   This is I think a result of mis-pasting from gdocs.  Replace with regular "